code/bignum/Hacl.Bignum4096.fst

```diff
@@ -0,0 +1,137 @@
+module Hacl.Bignum4096
+
+open FStar.Mul
+
+module BN = Hacl.Bignum
+module BM = Hacl.Bignum.Montgomery
+module BE = Hacl.Bignum.Exponentiation
+module BR = Hacl.Bignum.ModReduction
+module BI = Hacl.Bignum.ModInv
+
+#set-options "--z3rlimit 50 --fuel 0 --ifuel 0"
+
+let _ = assert_norm (4096ul = 64ul `FStar.UInt32.mul` 64ul)
+
+/// A note about the style of normalization used in this file. Normally,
```

Comment:
also outdated, and it looks like the assert above is perhaps already in the fsti?